modules/assim.batch/R/pda.emulator.R

```diff
@@ -211,15 +230,17 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
       for (i in seq_len(settings$assim.batch$n.knot)) {
         align.return <- pda.get.model.output(settings, run.ids[i], bety, inputs)
         model.out[[i]] <- align.return$model.out
-        if(!is.na(model.out[[i]])){
+        suppressWarnings(if(!is.na(model.out[[i]])){
           inputs <- align.return$inputs
-        } 
+        })
```

Comment:
Why are you suppressing warnings here? If it's because the conditional doesn't work, a handy function I've discovered recently is `isTRUE()`, which returns `TRUE` if and only if the inside is TRUE. 

```r
isTRUE(TRUE)
# TRUE
isTRUE(1 > NA)
# FALSE
isTRUE(1 > NULL)
# FALSE
isTRUE(NA)
# FALSE
isTRUE(NULL)
# FALSE
```